### DISCUSSION TITLE
dotSize vom Slider: In welcher Klasse ist diese hinterlegt?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1083] [Q&A]
Kann man den dotSize-Wert vom Slider per CSS beeinflussen?
Ich meine die kleinen:
![image](https://user-images.githubusercontent.com/84818934/136015723-34bfde78-2764-428b-bf79-9ba6d9bc694b.png)

**Comment by stony2k** [comment]
@Zefau Ich würde gerne an diese Frage anknüpfen. Vor allem auf dem Tablet wäre es interessant. Es scheint wohl mit dot-size oder dotsize zu gehen, aber wie genau muss das aussehen? Welche CSS Klasse wäre das?
![image](https://user-images.githubusercontent.com/16526471/143956317-b3224380-f04b-48b7-9cfd-11b8ebb00d53.png)

**Comment by mcuiobroker** [comment]
Vielleicht hilft das ein wenig:
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/styles/slider/widgets-id-.q-slider__thumb-container

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/styles/slider/widgets-id-role-slider-svg#groesse-aendern-slider-thumb

**Reply by stony2k** [reply]
> https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/styles/slider/widgets-id-role-slider-svg#groesse-aendern-slider-thumb

Wunderbar, habe Variante 2 aus deinem Link ausprobiert und es funktioniert wunderbar :)
![image](https://user-images.githubusercontent.com/16526471/144006455-17dc3d67-1d1f-45c5-b63e-027dfd49eecb.png)

**Comment by Bernd9000** [comment]
@mcuiobroker 

Meinst Du den ganzen Balken oder nur die Punkte ?

.q-slider__track-container--h {
     height: 10px;
}

**Reply by mcuiobroker** [reply]
Kannst du den passenden Link dazu reinstellen

**Reply by mcuiobroker** [reply]
Ach so ich meine nur die gelben Punkte.
Ich probier es morgen.System ist off.

**Comment by Bernd9000** [comment]
Die nennen sich wohl markers oder ticks.
Aber die größe konnte ich bisher auch noch nicht ändern.
Vielleicht hilft folgende Tipp:
[forum.quasar-framework.org](https://forum.quasar-framework.org/topic/7038/form-component-range-props-markers)
